### ISSUE TITLE
buildah from: fails if $XDG_RUNTIME_DIR does not exist

### ISSUE THREAD

**Issue by edsantiago** [projectatomic/buildah#618]
```
# buildah from centos
no such image "centos" in registry: image not known
# env -u XDG_RUNTIME_DIR buildah from centos
[succeeds]
# ls -l $XDG_RUNTIME_DIR
ls: cannot access '/run/user/0': No such file or directory
```
buildah-0.16-1524410543.gitd84f05a.fc27.x86_64

**Comment by rhatdan** [comment]
Do you have anything in /etc/containers/registries.conf

**Comment by edsantiago** [comment]
registries.conf is untouched:
```
# rpm -V atomic-registries
(no output)
```
This feels like it could be related to [libpod issue 659](https://github.com/projectatomic/libpod/issues/659)

**Comment by rhatdan** [comment]
Yup looks like the exact same problem.
@umohnani8 When you fix podman, we got the issue also.

**Comment by rhatdan** [comment]
@edsantiago good work keeping us on our toes.

**Comment by umohnani8** [comment]
Fixed by #646
